### PR TITLE
Fix batching test for PR 5977

### PR DESCRIPTION
**Pull request by asudarsa**
Signed-off-by: Arvind Sudarsanam <arvind.sudarsanam@intel.com>

### CODE REVIEW

File: SYCL/Plugin/level_zero_dynamic_batch_test.cpp [hunk 1]
```diff
@@ -166,7 +166,7 @@ int main(int argc, char *argv[]) {
   validate(Y1, Z7, M * N);
   validate(Y1, Z8, M * N);
 
-  for (size_t i = 0; i < 20; i++) {
+  for (size_t i = 0; i < 25; i++) {
```
Comment (againull) [inline SYCL/Plugin/level_zero_dynamic_batch_test.cpp:169]: Could you please clarify why this change is needed. Does test fail when 20 iterations? Or do you want to test some specific behaviour with 25 iterations?

Comment (againull) [reply SYCL/Plugin/level_zero_dynamic_batch_test.cpp:169]: Oh, I see from the comment to intel/llvm PR that test is failing when 20 iterations. Is it clear why it fails and is it expected to fail with 20 iterations?

Comment (asudarsa) [reply SYCL/Plugin/level_zero_dynamic_batch_test.cpp:169]: In PR 5977, we are changing the thresholds that control how batching is done. This change is expected to give some performance improvements. It does change the way the batch size is lowered/raised and that's what we test here. So, a change is needed here. Thanks.

**Review by againull — APPROVED** [review]
